Makefile

```diff
@@ -183,5 +183,423 @@ verify_version:
 
 verify_all: verify_pecl_file_definitions verify_version
 
-.PHONY: dist_clean clean all clang_format_check clang_format_fix install sudo_install test_c test_c_mem test_extension_ci test test_integration install_ini install_all \
+########################################################################################################################
+# TESTS
+########################################################################################################################
+REQUEST_INIT_HOOK := -d ddtrace.request_init_hook=$(PROJECT_ROOT)/bridge/dd_wrap_autoloader.php
+ENV_OVERRIDE := DD_TRACE_CLI_ENABLED=true
+
+### DDTrace tests ###
+TESTS_ROOT := ./tests
+COMPOSER := COMPOSER_MEMORY_LIMIT=-1 composer --no-interaction
+COMPOSER_TESTS := $(COMPOSER) --working-dir=$(TESTS_ROOT)
+PHPUNIT_OPTS := $(PHPUNIT_OPTS)
+PHPUNIT := $(TESTS_ROOT)/vendor/bin/phpunit $(PHPUNIT_OPTS) --config=$(TESTS_ROOT)/phpunit.xml
+
+TEST_INTEGRATIONS_54 := \
+	test_integrations_curl \
+	test_integrations_memcached \
+	test_integrations_mysqli \
+	test_integrations_pdo \
+	test_integrations_elasticsearch1 \
+	test_integrations_guzzle5 \
+	test_integrations_predis1
+
+TEST_WEB_54 := \
```

Comment:
It looks like `test_metrics` is missing from `TEST_WEB_54`, `TEST_WEB_55`, `TEST_WEB_56`, and `TEST_WEB_70`.

Reply:
good catch